frontend/src/app/components/post-modal/post-modal.component.ts

```diff
@@ -306,6 +328,58 @@ export class PostModalComponent {
     this.expandedUpvotesView = false;
   }
 
+  answerMultipleChoice() {
+    this.submitMultipleChoiceAnswer = true;
+    if (this.isMultipleChoiceSelected && this.selectedMultipleChoice) {
+      if (this.selectedMultipleChoice.correct) {
+        this.snackbarService.queueSnackbar('Correct Answer!');
+      } else {
+        this.snackbarService.queueSnackbar('Incorrect, Try Again.');
```

Comment:
Maybe either "Incorrect. Try again." (with a period after "Incorrect") or "Incorrect, please try again" (keeping the comma after "Incorrect"